### ISSUE TITLE
Perhaps BSD compat keywords

### ISSUE THREAD

**Issue by vbatts** [vbatts/go-mtree#43]
Like how keywords are annotated for being in the default set:
```bash
$ gomtree -list-keywords
Available keywords:
  md5
  rmd160digest                                                                                                                                                                                                       sha1
  xattr
  sha512
  sha512digest
  size  (default)
  link  (default)                                                                                                                                                                                                    rmd160
  ripemd160digest
  sha256digest
  sha1digest
  sha384digest                                                                                                                                                                                                       type  (default)
  time  (default)
  gid  (default)
  uname
  md5digest
  sha384
  tar_time
  uid  (default)
  nlink  (default)
  mode  (default)
  cksum
  sha256
```

Perhaps we should also annotate keywords for whether they are in the upstream mtree? Because upstream behavior is to fail if there is a keyword present that the tool does not recognize (i.e. `xattr` or `tar_time`).

**Comment by stephen679** [comment]
Sounds like a good idea. Speaking of documentation, we could start updating
the readme to include details about the archive compatibility

**Comment by vbatts** [comment]
Fixed by #45
